test/fixtures/plugin.filler/fill-line-boundary-end-span.json

```diff
@@ -39,6 +37,11 @@
                     "fill": "end",
                     "tension": 0
                 }
+            },
+            "plugins": {
```

Comment:
Is my assumption that this is needed to keep the filler plugin working correct?

Reply:
Yes. Also updates to the already supported universal way of disabling other core plugins. I have `filler: true` in some of these, which is not needed actually.